package-system/PhysX5/build_package_image.py

```diff
@@ -124,35 +131,55 @@ def cleanUpLibs(self, buildAsStaticLibs):
         static_bin_dir = self.workingDir / 'physx' / 'bin' / 'static'
         shared_bin_dir = self.workingDir / 'physx' / 'bin' / 'shared'
     
-        # Remove dynamic libraries repeated in static folders to save space.
-        # Also freeglut is not necessary for PhysX.
+        # Remove dynamic libraries, but copy some missing static libs from
+        # the shared builds into the static lib folder. Also freeglut is not
+        # necessary for PhysX.
         if self.platform == 'windows':
-            if buildAsStaticLibs:
+            if not buildAsStaticLibs:
+
+                for extra_static_lib_filename in ['LowLevel_static_64',
+                                                  'LowLevelAABB_static_64',
+                                                  'LowLevelDynamics_static_64',
+                                                  'PhysXTask_static_64',
+                                                  'SceneQuery_static_64',
+                                                  'SimulationController_static_64']:
+
+                    for extra_static_lib_ext in ['.lib', '.pdb']:
```

Comment:
Are we copying the .pdbs? Do we want to do that? Are they really big? Maybe we could output those to a separate place where they can be zipped and uploaded separate?

Reply:
This is a common scenario with a lot of these packages. Lets hold off on that until we come up with a general strategy for separate distribution of debug symbols for 3P packages and O3DE